plumbing/storer/storer.go

```diff
@@ -13,3 +13,13 @@ type Initializer interface {
 	// any.
 	Init() error
 }
+
+// Options holds configuration for the storage.
+type Options struct {
+	// ExclusiveAccess means that the filesystem is not modified externally
```

Comment:
Since this is options strictly related to a one implementation I don't believe this is the right place to put this options, this options should be pass and used only to the file system implementation